campers.html

```diff
@@ -18,29 +18,22 @@
       <div class="include" data-include="header">
       </div>
       <div id="content" >
-        <div class="camper_blogs">
+        <div class="campers_page">
           <div class="subtitle">
-            <h2>Blogs and Projects</h2>
+            <div class="blog_toggle" id="blog_toggle"><h2>Blogs</h2></div><div class="projects_toggle" id="projects_toggle"><h2>Projects</h2></div>
```

Comment:
Yes. "Blogs" and "Projects" end up on different lines otherwise. I think HTML is registering a space when there isn't one.